e2e/scenario/validators.go

```diff
@@ -72,6 +72,24 @@ func NvidiaSMINotInstalledValidator() *LiveVMValidator {
 	}
 }
 
+func NvidiaSMIInstalledValidator() *LiveVMValidator {
```

Comment:
I wanted to combine installed/not installed validators for smi somehow, but I think it's cleaner to just leave these like that for now.